### PR TITLE
[1LP][RFR] Fixing Container Images perform_smartstate_analysis with correct task name

### PR DESCRIPTION
**Pull request by brahmanim**
Following cfme/configure/tasks.py update that has been done,

I had to fix cfme/containers/image.py perform_smartstate_analysis with correct task name,
That includes "Container Image Analysis: " prefix on task name.

Also, an update regarding  perform_smartstate_analysis_multiple_images,
update the following functions regarding TasksCollection:
is_finished, is_successfully_finished and wait_for_finished.

{{pytest: cfme/tests/containers/test_containers_smartstate_analysis.py -v --use-provider ocp-37-hawk }}

### CODE REVIEW

File: cfme/containers/image.py [hunk 1]
```diff
@@ -203,24 +203,26 @@ def perform_smartstate_analysis_multiple_images(
         # task_name change from str to regular expression
         # the str compile on tasks module
         image_enities_names = []
+        delay = 5
+        silent_failure = True
```
Comment (lkhomenk) [inline cfme/containers/image.py]: QUESTION: why are there hardcoded?

Comment (brahmanim) [reply cfme/containers/image.py]: No special reason, Just to understand your question, what is your suggestion?
Are you suggest to use this variable values directly on the function call?

Comment (lkhomenk) [reply cfme/containers/image.py]: @brahmanim  delay is already defaulted to 5 - https://github.com/ManageIQ/integration_tests/pull/7337/files/6bc76133010fd1e4abcf5f2273c39d001e4639c8#diff-1fe1d4bbe572be5ebf426e918f68f31fR261
and `silent_failure` better to pass as attribute directly into `is_successfully_finished`. I don't see a need in additional local variables

Comment (brahmanim) [reply cfme/containers/image.py]: OK, just wanted to be sure before I make changes.

Comment (brahmanim) [reply cfme/containers/image.py]: Since i Will move back to WIP, Are you have any forther comments?

Comment (lkhomenk) [reply cfme/containers/image.py]: nothing else ;) PR is fine. and even more - green PRT is here!

**Comment by brahmanim** [comment]
@lkhomenk Local Variables, delay & silent_failure, removed.

File: cfme/containers/image.py [hunk 2]
```diff
@@ -209,18 +209,18 @@ def perform_smartstate_analysis_multiple_images(
         images_view.toolbar.configuration.item_select(
             'Perform SmartState Analysis', handle_alert=True)
         for image_entity in image_entities:
-            image_enities_names.append(image_entity.name)
+            image_enities_names.append("Container Image Analysis: '{}'".format(image_entity.name))
```
Comment (izapolsk) [inline cfme/containers/image.py]: REQUIRED: typo

File: cfme/configure/tasks.py [hunk 3]
```diff
@@ -226,17 +227,24 @@ def delete_all(self):
...
+            if row.task_name.text in tasks and row.state.text.lower() != "finished":
                 return False
         return True
 
     def is_successfully_finished(self, silent_failure=False, *tasks):
         view = navigate_to(self, self.tab)
         tab_view = getattr(view.tabs, self.tab.lower())
         rows = []
+        # expected_status support also regular expression pattern
+        expected_status = re.compile('finished', re.IGNORECASE)
```
Comment (izapolsk) [inline cfme/configure/tasks.py]: OPTIONAL: I'd rather use "in" or "contains" but this also is ok.

Comment (izapolsk) [reply cfme/configure/tasks.py]: yeah, there is no good way to avoid using regexp

**Pull request merged**